### DISCUSSION TITLE
Change Emulated CPU speeds on the fly?

### DISCUSSION THREAD

**Discussion by KoghaandSooga** [86Box/86Box#3783] [Q&A] [answered]
Someone asked this over here on Reddit but never got answered so I have wanted to know for a while too. https://www.reddit.com/r/86box/comments/17gts8t/is_there_a_way_to_change_emulated_cpu_speed_on/

On the other emulator as long as you stick to the same motherboard you can change the emulated CPU on the fly so I'm curious why that isn't the case here on 86box? Each CPU speed requires a hard reboot which means if you as some say want to lower the speed for certain dos games or try things out at various speed levels it can get old pretty quick yet in some cases 86box has more features.

 Is there something dynamically different about the coding or is it human laziness that's the issue at stake or something else entirely? 😕

**Comment by OBattler** [comment, marked as answer]
I removed the feature because it would require a more reliable framework for changing timings on the fly before it's reimplemented. This is also why turbo is not yet implemented. But it's certainly in the plans, at least as a feature behind cpu_override.

**Reply by KoghaandSooga** [reply]
Why do some of my early childhood games (which uses Quicktime) lag hard while 3D games that are more demanding actually play on a much higher emulated CPU? I can't play Where In The World Is Carmen Sandiego (The one with the characters walking around) on Pentium II while I can play Streets of Sim City on almost the highest CPU.
